packages/provider/src/TiptapCollabProvider.ts

```diff
@@ -17,6 +19,15 @@ const defaultDeleteCommentOptions: DeleteCommentOptions = {
   deleteThread: false,
 }
 
+const defaultGetThreadsOptions: GetThreadsOptions = {
+  types: ['undeleted'],
```

Comment:
undeleted isn't a word and doesn't sound right.

How about archived & not archived. Don't need two states, when people get, it should only ever be not archived. And asking for archived means you want archived too.

Like `includeArchived?: true`

Reply:
Or includeDeleted if you wanna stick to that naming